docs: markdown content: Add section about how to customize the plugins (#2632)
withastro/docs: src/pages/en/guides/markdown-content.mdx

}
```

#### Customizing a plugin

> In order to customize your plugins, provide the options object in a nested array. The example below adds the [`behavior` option to the `rehype-autolink-headings` plugin](https://github.com/rehypejs/rehype-autolink-headings#optionsbehavior) in order to add the anchor tag after the headline text.

Reviewer: ```suggestion
In order to customize a plugin, provide an options object after it in a nested array. The example below adds the [`behavior` option to the `rehype-autolink-headings` plugin](https://github.com/rehypejs/rehype-autolink-headings#optionsbehavior) in order to add the anchor tag after the headline text.
```
